packages/livekit-server-sdk/src/IngressClient.ts

```diff
@@ -67,11 +67,11 @@ export interface UpdateIngressOptions {
    */
   name: string;
   /**
-   * name of the room to send media to. optional
+   * name of the room to send media to.
```

Comment:
this is optional right? on updates.

Reply:
Everything is optional in the dictionary in the sense that parameters left out will not be updated. It's odd to single out these 2 parameters. We could flag all parameters as optional, but this seems redundant on a parameter called "UpdateIngressOptions".

Reply:
got it.. makes sense.